web/backend/src/controllers/dela.ts

```diff
@@ -174,6 +176,29 @@ delaRouter.use('/services/shuffle/:formID', (req, res, next) => {
   }
   next();
 });
+
+delaRouter.post('/forms/:formID/vote', (req, res) => {
+  if (!req.session.userId) {
+    res.status(401).send('Authentication required!');
+    return;
+  }
+  if (!isAuthorized(req.session.userId, req.params.formID, PERMISSIONS.ACTIONS.VOTE)) {
```

Comment:
General question: is it possible for a user to spoof the `userId`, or is that some internal variable that is protected?

Reply:
I don't remember - are the session cookies signed, and the signature verified ? (ExpressJS allows this, I'm not sure we use that option though)

If not, they can be spoofed.

Reply:
I'll open an issue.